src/System.Private.ServiceModel/src/System/ServiceModel/Description/ClientCredentials.cs

```diff
@@ -18,6 +18,7 @@ public class ClientCredentials : SecurityCredentialsManager, IEndpointBehavior
         private WindowsClientCredential _windows;
         private HttpDigestClientCredential _httpDigest;
         private bool _isReadOnly;
+        private bool _useIdentityConfiguration = false;
```

Comment:
We don't have config. I would remove this field, and on the property getter always return false and have the setter throw if true is passed, otherwise ignore.